examples/kitchen-sink.yaml

```diff
@@ -426,3 +426,68 @@ resource:
         - process.command_args
   # Configure the resource schema URL.
   schema_url: https://opentelemetry.io/schemas/1.16.0
+# Configure instrumentation.
+instrumentation:
+  # Configure general SemConv options that may apply to multiple languages and instrumentations.
+  general:
+    # Configure instrumentations following the peer semantic conventions.
+    # 
+    # See peer semantic conventions: https://opentelemetry.io/docs/specs/semconv/attributes-registry/peer/
+    peer:
+      # Configure the service mapping for instrumentations following peer.service semantic conventions.
+      # 
+      # See peer.service semantic conventions: https://opentelemetry.io/docs/specs/semconv/general/attributes/#general-remote-service-attributes
+      service-mapping:
+        - peer: 1.2.3.4
+          service: FooService
+        - peer: 2.3.4.5
+          service: BarService
+    # Configure instrumentations following the http semantic conventions.
+    # 
+    # See http semantic conventions: https://opentelemetry.io/docs/specs/semconv/http/
+    http:
+      # Configure instrumentations following the http client semantic conventions.
+      client:
+        request:
+          capture-headers:
+            - X-Foo
+        response:
+          capture-headers:
+            - X-Foo
+      # Configure instrumentations following the http server semantic conventions.
+      server:
+        request:
+          capture-headers:
+            - X-Bar
+        response:
+          capture-headers:
+            - X-Bar
+    # Configure instrumentations following the db semantic conventions.
+    #
+    # See db semantic conventions: https://opentelemetry.io/docs/specs/semconv/database/
+    db:
+      # Configure database statement sanitization.
+      #
+      # See https://opentelemetry.io/docs/specs/semconv/database/database-spans/#common-attributes
+      statement-sanitizer:
```

Comment:
I think this is another instance of "hinted at by spec", in a footnote (5) for [db.statement](https://opentelemetry.io/docs/specs/semconv/database/database-spans/#common-attributes):

> Should be collected by default only if there is sanitization that excludes sensitive information.

It doesn't mention whether users should be able to configure the behavior, though. In which case, maybe it gets the chop?